packages/typescript/integrations/openai/src/index.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * Main tracedOpenAI wrapper function for MLflow tracing integration
+ */
+
+import { withSpan } from '../../../src/core/api';
+import { SpanType } from '../../../src/core/constants';
```

Comment:
not sure if this was my own local issue, but when building this package locally it seems to build the parent package. changing to import the package seems to fix (along with other changes to tsconfig)

```suggestion
import { withSpan } from 'mlflow-tracing';
import { SpanType } from 'mlflow-tracing';
```

<img width="182" alt="Screenshot 2025-07-09 at 2 56 30 PM" src="https://github.com/user-attachments/assets/885b1ac1-f1d9-448f-b463-0fb99d3cc797" />